app/command_test.go

```diff
@@ -75,6 +75,45 @@ func TestCreateCommandPost(t *testing.T) {
 	}
 }
 
+func TestExecuteCommand(t *testing.T) {
+	th := Setup(t).InitBasic()
+	defer th.TearDown()
+
+	// Testing slash command message is correctly parsed
+	TestCases := []string{
```

Comment:
A few code formatting suggestions. I think the tests would be easier to manage as a map, since the input and expected output are closer together.
```go
testCases := map[string]string {
    "/code happy path": "    happy path",
    // ...
}
```

Reply:
you are absolutely right